Sources/Commands/SwiftTool.swift

```diff
@@ -152,6 +160,14 @@ public class SwiftTool<Options: ToolOptions> {
                 // should be moved to file system APIs with currentWorkingDirectory.
                 try chdir(dir.asString)
             }
+
+            let processSet = ProcessSet()
+            interruptHandler = try InterruptHandler {
+                // Terminate all processes on receiving an interrupt signal.
+                processSet.terminate()
```

Comment:
One random request, as a follow on: when we are interrupted via SIGINT, we should die via SIGINT, because of: https://www.cons.org/cracauer/sigint.html